modules/metering-install-verify.adoc

```diff
@@ -28,7 +28,7 @@ $ oc --namespace openshift-metering get csv
 [source,terminal,subs="attributes+"]
 ----
 NAME                                           DISPLAY                  VERSION                 REPLACES   PHASE
-elasticsearch-operator.{product-version}.0-202006231303.p0   Elasticsearch Operator   {product-version}.0-202006231303.p0              Succeeded
+elasticsearch-operator.{product-version}.0-202006231303.p0   OpenShift Elasticsearch Operator   {product-version}.0-202006231303.p0              Succeeded
```

Comment:
Have we verified that this is the actual output?

Reply:
> Have we verified that this is the actual output?

Thanks for checking. I'll verify the output and update this PR accordingly.

Reply:
@kabirbhartiRH would you verify this output for me?

Reply:
@rolfedh The actual output is like 

oc get csv -n openshift-logging
NAME                              DISPLAY                            VERSION    REPLACES   PHASE
cluster-logging.5.1.0-78          Cluster Logging                    5.1.0-78              Succeeded
elasticsearch-operator.5.1.0-99   OpenShift Elasticsearch Operator   5.1.0-99              Succeeded